yaml/05_search_summary_webex.yaml

```diff
@@ -0,0 +1,12 @@
+Title: Blaze Bot Transcription 
```

Comment:
Can we change this to BLAZE Transcript Analysis? Since I think the embedded meetings app will use the same yaml file?

Reply:
Changed